plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTestHelper.java

```diff
@@ -46,4 +53,45 @@ public static Dictionary createAssetPackLocationsDictionary() {
     testDict.put("location2", innerDict2);
     return testDict;
   }
+
```

Comment:
Refactored duplicate code into factory methods in PlayAssetDeliveryTestHelper class.